Formula/openssl.rb

```diff
@@ -57,10 +57,15 @@ def install
       arch = Hardware::CPU.arch_32_bit
     end
 
+    depend_args = []
```

Comment:
Could you add a comment above this linking to the relevant issue and explaining why it shouldn't be deleted (i.e. we need to keep 10.5 working for building portable Ruby).